scripts/repro_matrix/utils.py

```diff
@@ -66,3 +67,39 @@ def run_eval_and_return_metric(metric, eval_key, defs, runfile):
 
     return 0.0
 
+def run_dpr_retrieval_eval_and_return_metric(defs, json_file):
+    """Generate dpr retrieval evaluation scores
+
+    Args:
+        defs: topk definitions (e.g., '--topk 5 20')
+        json_file: dpr retrieval json file
+
+    Returns:
+        topk: a dictionary of topk scores (e.g., {"Top5": <score>})
+    """    
+    eval_cmd = f'python -m pyserini.eval.evaluate_dpr_retrieval --retrieval {json_file} {defs} '
+    eval_stdout, eval_stderr = run_command(eval_cmd)
+    topk = {}
+    for line in eval_stdout.split('\n'):
+        parts = line.split('\t')
+        if len(parts) == 2 and 'accuracy' in parts[1]:
+            topk.update({parts[0]:round(float(parts[1][10:])*100, 4)})
+    return topk
+
+
+def convert_trec_run_to_dpr_retrieval_json(topics,index,runfile,output):
+    """Convert trec runfile to dpr retrieval json file
+
+    Args:
+        topics: topics field
+        index: index field 
+        runfile: input runfile
+        output: output jsonfile
+
+    Returns:
+        dummy value: 0.0
+    """    
+    cmd = f"python -m pyserini.eval.convert_trec_run_to_dpr_retrieval_run --topics {topics} --index {index} --input {runfile} --output {output}"
+    os.system(cmd)
+    return 0.0
```

Comment:
How about we return the value that `os.system` returns? Corresponds to the exit status of the process...